src/cmd/app.rs

```diff
@@ -51,22 +134,125 @@ pub enum Potency {
     Destroy,
     SuperUser,
 }
-pub fn run_command(a: App) {
+
+impl Potency {
+    fn to_string(&self) -> String {
+        match self {
+            Potency::Append => "Append".to_string(),
+            Potency::Destroy => "Destroy".to_string(),
+            Potency::SuperUser => "Super_User".to_string(),
+        }
+    }
+}
+
+pub fn run_command(a: App) -> Result<()> {
     match a.command {
         AppCommands::Delegate {
-            app_name: _,
-            did: _,
-            potency: _,
-            lifetime: _,
-            quiet: _,
+            app_name,
+            did,
+            lifetime,
+            potency,
+            quiet,
+            verbose,
+            remote,
         } => {
-            todo!("delegate")
+            let args = prepare_args(&HashMap::from([
+                ("-a", app_name.as_ref()),
+                ("-d", did.as_ref()),
+                ("-l", Some(lifetime.to_string()).as_ref()),
+                ("-p", Some(potency.to_string()).as_ref()),
+            ]));
+            let remote = prepare_args(&HashMap::from([("-R", remote.as_ref())]));
+
+            // N.B. The wrapped app delegate command does not accept verbose
+            let flags = prepare_flags(&HashMap::from([("-q", quiet)]));
+
+            Command::new("fission")
+                .args(["app", "delegate"])
+                .args(args)
+                .args(remote)
```

Comment:
The remote arg must come after the other args, but before flags for compatibility with the existing CLI.